composer.json

```diff
@@ -90,7 +90,8 @@
     },
     "patches": {
       "phpunit/phpunit-mock-objects": {
-        "Fix ReflectionParameter warnings on PHP 8": "patches/phpunit-mock-objects.patch"
+        "Fix ReflectionParameter warnings on PHP 8": "patches/phpunit-mock-objects.patch",
+        "Remove 'match' keyword from uses <https://github.com/sebastianbergmann/phpunit/issues/4373>": "patches/remove-match-keyword.patch"
```

Comment:
Regarding https://github.com/ampproject/amp-wp/pull/7088#issuecomment-1116153312, I'm curious how you were able to find the solution.

Reply:
This was added in [613eca](https://github.com/ampproject/amp-wp/pull/7088/commits/dcd3dfeef0dd942c160262f4c303566044613eca) commit. And for that, I took some references from [here](https://github.com/sebastianbergmann/phpunit/pull/4374/files).

So it basically applies changes of [this](https://github.com/sebastianbergmann/phpunit/pull/4374/files) PR into an installed version of `phpunit/phpunit-mock-objects` package.